homeassistant/components/media_player/plex.py

```diff
@@ -192,6 +200,20 @@ def update_devices():
             client.set_availability(client.machine_identifier
                                     in available_client_ids)
 
+            if not config.get(CONF_REMOVE_UNAVAILABLE_CLIENTS) \
+                    or client.available:
+                continue
+
+            if (dt.now() - client.marked_unavailable).total_seconds() >= \
```

Comment:
No need to use `total_seconds()` for both timedeltas. Just compare the two timedeltas directly.

Reply:
/sigh.. doh.. good point